saleor/product/migrations/tasks/saleor3_9.py

```diff
@@ -17,14 +17,14 @@ def set_description_plaintext(qs: QuerySet[Product]):
 
 
 @app.task
-def set_description_plaintext_task():
+def set_description_plaintext_task(last_pk: int = 0):
     products = Product.objects.filter(
-        Q(description_plaintext="") & ~Q(description=None)
+        Q(description_plaintext="", pk__gt=last_pk) & ~Q(description=None)
     ).order_by("-pk")
 
     ids = products.values_list("pk", flat=True)[:2000]
     qs = Product.objects.filter(pk__in=ids)
 
     if ids:
         set_description_plaintext(qs)
-        set_description_plaintext_task.delay()
+        set_description_plaintext_task.delay(qs.last().pk)
```

Comment:
And probably `ids[-1]` will be safer if applied what I mention above.

Reply:
It won't work as slices does not provide that functionality

Reply:
`max(ids)` should work, though, no? The QuerySet is already evaluated.